runtime/manta/src/lib.rs

```diff
@@ -203,32 +184,22 @@ impl Contains<Call> for MantaFilter {
             return true;
         }
 
-        if pallet_tx_pause::PausedTransactionFilter::<Runtime>::contains(call) {
-            // no paused call
-            return false;
-        }
-
         #[allow(clippy::match_like_matches_macro)]
         // keep CallFilter with explicit true/false for documentation
         match call {
-            // Explicitly DISALLOWED calls
-            // Filter Utility to prevent users from setting keys and selecting collator for parachain (couldn't use now).
-            Call::Utility(_)
-            // Filter Session and CollatorSelection to prevent users from utility operation.
-            | Call::Session(_)
-            | Call::CollatorSelection(_)
-            | Call::XcmpQueue(_) | Call::PolkadotXcm(_) | Call::DmpQueue(_) => false, // Filter XCM pallets
-
+            // Explicitly DISALLOWED calls ( Pallet user extrinsics we don't want used WITH REASONING )
             // Explicitly ALLOWED calls
             | Call::Authorship(_)
             // Sudo also cannot be filtered because it is used in runtime upgrade.
             | Call::Sudo(_)
             | Call::Multisig(_)
-            | Call::TransactionPause(_)
-            | Call::Balances(_) => true,
+            | Call::AuthorInherent(pallet_author_inherent::Call::kick_off_authorship_validation {..}) // executes unsigned on every block
+            | Call::Balances(_)
```

Comment:
`Balances` is allowed?

Reply:
I suppose we don't need to do it now, but this way we won't need another runtime upgrade after distributing tokens to allow users to transfer them.

I don't see a downside to having it enabled since nobody will have a balance to transfer anyway